content/blog/2016/2016-09-27-jenkins-world-2016-wrap-up-scaling.adoc

```diff
@@ -0,0 +1,141 @@
+link:https://github.com/carlossg[Carlos Sanchez] of
+link:http://cloudbees.com[CloudBees] discussed
+link:https://www.cloudbees.com/scaling-jenkins-docker-swarm-kubernetes-or-mesos[*Scaling Jenkins with Docker: Swarm, Kubernetes or Mesos?*]
+He compared various Docker Swarm, Apache Mesos, and Kubernetes in terms of their
+ability to dynamically scale in Jenkins by running jobs inside containers.
+He also discussed the pros and cons, best practices, level of Jenkins support for each
+of these technologies.
+
+link:https://www.cloudbees.com/sites/default/files/2016-jenkins-world-scaling_jenkins_with_docker_swarm_kubernetes_or_mesos_.pdf[*Download Presentation*]
+
+---
+
+link:https://www.cloudbees.com/so-you-want-build-worlds-biggest-jenkins-cluster[*So, You Want to Build the World's Biggest Jenkins Cluster?*]
+link:https://github.com/stephenc[Stephen Connolly] of
+link:http://cloudbees.com[CloudBees] showed all about it.  He started
+real world results realized by Jenkins users who have built large clusters.
+Next, he showed experiments around scaling some individual sub-components of Jenkins in
+isolation to see what challenges have been faced when integrated. Finally,
+he arrived at recipes for building Jenkins clusters with different scaling capabilities and
+making existing Jenkins clusters more efficient.
+
+link:https://www.cloudbees.com/sites/default/files/2016-jenkins-world-soyouwanttobuildtheworldslargestjenkinscluster_final.pdf[*Download Presentation*]
+
+---
+
+image:/images/post-images/scaling-jenkins-at-jenkinsworld/splunk-logo-300x100.gif[role=right]
+
+
+link:https://www.linkedin.com/in/bill-houston-4909412[Bill Houston] and
+link:https://www.linkedin.com/in/aliraza82[Ali Raza] of
+link:https://www.splunk.com/[Splunk]
+gave a talk in two parts,
+link:https://www.cloudbees.com/jenkins-splunk-and-splunking-jenkins[*Jenkins at Splunk and Splunking Jenkins*]
+They first showed how Splunk uses Jenkins to implement their end-to-end CI system.
+They discussed features and design goals, challenges they encountered, and how they addressed
+these challenges.
+In the second part, they showed how to use the Jenkin Splunk plugin to gather
+test results and Jenkins environment data, and deliver it to a Splunk indexer for analysis and presentation.
+This included a demonstration.
+
+link:https://www.cloudbees.com/sites/default/files/2016-jenkins-world-jenkins_at_splunk.pdf[*Download Presentation*]
+
+---
+
+image:/images/post-images/scaling-jenkins-at-jenkinsworld/272px-Google_2015_logo.svg.png[role=left]
+
+link:https://www.linkedin.com/in/david-hoover-4265245[David Hoover] of
+link:https://www.google.com/about/[Google] talked about
+link:https://www.cloudbees.com/jenkins-inside-google[*Jenkins inside Google*].
+Last year, they presented their initial investigations and stress testing as they
+prepared to deploy a large-scale Jenkins installation at Google. Now, with a
+year of real-world use under their belts, they returned to present how their
```

Comment:
I'm not a linguist but I think "they returned to present how their" should be "they returned to present on how their"